addons/mail/static/src/core/common/message.xml

```diff
@@ -184,4 +186,9 @@
     </div>
 </t>
 
+<t t-name="mail.Message.pendingStatus">
+    <button t-if="message.postFailRedo" class="btn p-0" title="Failed to post the message. Click to retry" t-on-click="() => message.postFailRedo?.()"><i class="fa fa-fw fa-warning text-warning"/></button>
```

Comment:
The `?` on `postFailRedo` inside the `onclick` is useless because it should already be guarded by the `t-if`

Reply:
The template is based on rendering, it could still be visible on UI so user could click even when this is false